<?php

namespace app\index\model\b2b_tour;
use think\Exception;
use think\Model;
use think\Db;
class B2bTourItinerary extends Model{
    //protected $connection = ['database' => 'erp'];
    protected $table = 'b2b_tour_itinerary';
    public function initialize()
    {
    	parent::initialize();
    
    }


    public function addB2bTourItinerary($params){

        $data['btb_tour_id'] = $params['btb_tour_id'];
        $data['the_day'] = $params['the_day'];
        $data['hotel_cn'] = $params['hotel_cn'];
        $data['hotel_en'] = $params['hotel_en'];
        $data['info_cn'] = $params['info_cn'];
        $data['info_en'] = $params['info_en'];
    	$data['status'] = $params['status'];

    	try{
    		Db::name('b2b_tour_itinerary')->insert($data);
    	} catch (\Exception $e) {

            $result = $e->getMessage();
    		\think\Response::create(['code' => '400', 'msg' =>$result], 'json')->send();
    		exit();
    	}

    }

}